stories/02-configuration/05-Multisite-Support.stories.mdx

```diff
@@ -0,0 +1,39 @@
+<Meta title="Configuration|Multisite Support" />
+
+# Multisite Support
+This section outlines how to configure Irving to support multiple different sites using a single instance.
+
+## Overview
+An single instance of Irving is capable (since version 5.2.0) of serving multiple different URLs. Irving will query a different backend API root URL conditionally based on the incoming hostname in the HTTP request. Each backend host should be configured to be compatible with the same component map.
```

Comment:
```suggestion
A single instance of Irving is capable (since version 5.2.0) of serving multiple different URLs. Irving will query a different backend API root URL conditionally based on the incoming hostname in the HTTP request. Each backend host should be configured to be compatible with the same component map.
```